README.md

```diff
@@ -48,8 +48,8 @@ Collection of examples that solve specific problems using small pieces of code.
 
 ### FastAPI Examples
 
-- [FastAPI Base Example](./examples/fastapi-base/README.md)
-- [FastAPI CRUD Example](./examples/fastapi-crud/README.md)
+- [FastAPI Base Example](./examples/fastapi-base/README.md) (FastAPI, Faker, Factory-Boy, Pydantic, starlette)
+- [FastAPI CRUD Example](./examples/fastapi-crud/README.md) (FastAPI, Alembic, Dockerfile, Docker-compose, Pydantic, SQLAlchemy, PostgreSQL)
```

Comment:
can you also include a small description of what it is? the format can be

```
[Example Name](example link) - Short description. Technologies list.
```

Reply:
Ok sounds good

Reply:
Done, but notice that I had to make some changes in workflows and pre-commit